File: LunchApp/Data/Models/UserLunch.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace Data.Models
{
    public class UserLunch
    {
        public int UserLunchId { get; set; }
        public int UserId { get; set; }

        [ForeignKey("UserId")]
        public User User { get; set; }

        public string FilePath { get; set; }

        public int MenuId { get; set; }
        [ForeignKey("MenuId")]
        public Menu Menu { get; set; }

        public bool Submitted { get; set; }
        public bool Editable { get; set; }

        public DateTime SubmitionDate { get; set; }
        public DateTime CreationDate { get; set; }

        [ForeignKey("UserLunchId")]
        public ICollection<UserLunchItem> UserLunchItems { get; set; }
    }
}

File: LunchApp/lunch-app/src/app/models/userLunch/userLunch.ts
import { User } from "app/models/user/user";
import { UserLunchSection } from "app/models/userLunch/userLunchSection";

export class UserLunch {
    user: User;
    menuId:number;
    userLunchId:number;
    sections:UserLunchSection[];
    lunchDate:string;
    price: number;
    editable: boolean;
    selectedForOrder: boolean;
 }
